src/lib/core/fiber.h

```diff
@@ -429,6 +430,49 @@ fiber_clock64(void);
 API_EXPORT void
 fiber_reschedule(void);
 
+/**
+ * Set fiber name.
+ * @param fiber Target fiber, if it's NULL the current fiber is used.
+ * @param name A new name of @a fiber.
+ * @param len Length of the string pointed to by @a name.
+ */
+API_EXPORT void
+fiber_set_name_n(struct fiber *fiber, const char *name, uint32_t len);
+
+/**
+ * Get fiber name.
+ * @param fiber Target fiber, if it's NULL the current fiber is used.
+ * @return pointer to a nul-terminated string.
+ */
+API_EXPORT const char *
+fiber_get_name(struct fiber *fiber);
```

Comment:
I suggest to make all getters take `const struct fiber *` instead of a mutable pointer.

Reply:
Done